cmd/handler-api.go

```diff
@@ -103,7 +103,8 @@ func availableMemory() (available uint64) {
 	if err != nil {
 		return
 	}
-	available = memStats.Available / 2
+	// A valid value is available return its 75%
+	available = memStats.Available * (3 / 4)
```

Comment:
```suggestion
	available = (memStats.Available * 3) / 4
```
parenthesis for clarity